apps/core/src/migration/migration.service.ts

```diff
@@ -210,6 +235,22 @@ export class MigrationService {
 
         // Create IoT Application dashboards
         await this.createApplicationDashboards(convertedDashboards);
+
+        if (this.parsingErrors.length !== 0) {
+          let errorMessage =
+            'error parsing dashboard definitions for SiteWise Monitor dashboard(s): ';
+          for (const error of this.parsingErrors) {
+            errorMessage += `${error.message},`;
+          }
+          // Remove trailing comma
+          errorMessage = errorMessage.slice(0, -1);
```

Comment:
nit: a `string.join(...)` can replace those few lines

Reply:
Good call, updated in new commit push